packages/admin-stories/src/docs/components/Table/01_Basics.stories.mdx

```diff
@@ -0,0 +1,169 @@
+import { Meta, Canvas, Story, Source } from "@storybook/addon-docs";
+
+import dedent from "ts-dedent";
+
+<Meta title="Docs/Components/Table/Basics" />
+
+# Table
+
+> Tables display information in a way that's easy to scan, so that users can look for patterns and insights.
+>
+> \- [MUI Docs](https://mui.com/components/tables/)
+
+The Comet Admin `Table` internally uses [Material UI's `Table` component](https://v4.mui.com/components/tables/) and its [associated components](https://v4.mui.com/components/tables/#api).
```

Comment:
> I would add an intro section where you mention the differences between Comet Admin Table and MUI Table (and the fact that MUI Table is used internally)

@nsams I added some intro text and a section about "Differences between Comet Admin `Table` and `MuiTable`". Is this what you had in mind?

Reply:
perfect.

(later we should mention DataGrid here)